src/content/docs/en/guides/client-side-routing.mdx

```diff
@@ -44,7 +42,7 @@ Use client-side routing in Astro with the `<ViewTransitions />` routing componen
 
 ## Client-side Routing on a Page
 
-Import the `<ViewTransitions />` component and place it inside of a page's head to opt that page in to client-side routing on a per-page basis. You can additionally control the animation of page elements during navigation with [transition directives](#transition-directives):
+Import the `<ViewTransitions />` component and place it inside of a page's `<head>` to opt that page in to client-side routing on a per-page basis. You can then control the animation of page elements during navigation with [transition directives](#transition-directives):
```

Comment:
Guess here it should be "into"?
```suggestion
Import the `<ViewTransitions />` component and place it inside of a page's `<head>` to opt that page into client-side routing on a per-page basis. You can then control the animation of page elements during navigation with [transition directives](#transition-directives):
```

Reply:
Interesting question! I was thinking of it from the "opt in" phrasing perspective, and while we probably could get away with either one, and language evolving yadda yadda yadda... I think *technically* and *pedantically* mine is considered traditionally correct? And maybe the jury is out on universally accepting "into"?

We're not moving "into" client-side routing. We're opting in... to what? To client-side routing? A quick check of a few sources isn't conclusive re: absolutely correct, but everyone's take seems to lean towards mine.  :smile:

Reply:
Yeah, this one is confusing, I'm good with going with your version!

Reply:
I don't think I understand this thread quite... happy with whatever you all decide here.